website/lab/02/index.mdx

```diff
@@ -539,11 +539,20 @@ pin.set_high();
 pin.set_low();
 ``` 
 
+</TabItem>
+
+  <TabItem value="stm32u5" label="STM32 Nucleo-U545RE-Q" default>
+use embassy_stm32::gpio::{Input, Pull};
+let mut pin = Input::new(PxN, Pull::Up);
+
+let is_high = pin.is_high();
+let is_low = pin.is_low();
```

Comment:
```suggestion

```rust
use embassy_stm32::gpio::{Input, Pull};

// initialize PXn
//   - replace X with the port letter(A, B, C, D, E, etc.)
//   - n with the pin number
//   ... see the datasheet, e.g. PA5 
// and activate its internal pull up resistor.
let pin = Input::new(peripherals.PXn, Pull::Up);

if pin.is_high() {
    // Do something if the pin value is HIGH (1)
} else {
    // Do something if the pin value if LOW (0)
}
```

Reply:
Please place this in the *Configure GPIO Input* section.